### PR TITLE
Subaru: rename legacy to preglobal

### PR DESCRIPTION
**Pull request by jnewb1**
**Description** "Legacy" is a confusing name because there is a car called the "Subaru Legacy", "Preglobal" is a better name.

**Verification** Should be functionally equivalent, just name changes

https://github.com/commaai/cereal/pull/445
https://github.com/commaai/panda/pull/1415

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [submodule "panda"]
   path = panda
-  url = ../../commaai/panda.git
+  url = ../../jnewb1/panda.git
```
Comment (adeebshihadeh) [inline .gitmodules:3]: merged the other PRs, can you revert this?

**Review by adeebshihadeh — CHANGES_REQUESTED** [review]

File: .gitmodules (hunk 1 above)
Comment (jnewb1) [reply .gitmodules]: done

**Comment by adeebshihadeh** [comment]
The submodules will have to be updated to the versions on master.

**Comment by jnewb1** [comment]
> The submodules will have to be updated to the versions on master.

done

**Comment by adeebshihadeh** [comment]
Nice cleanup!

**Pull request merged**